.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Every Other Monday
+on:
+  workflow_dispatch:
+  # @TODO: byweekly check
+  # Run action at 16:15 PM on Monday (UTC)
+  # schedule:
+  #   - cron: '15 16 * * 1'
+
+# Sets the GITHUB_TOKEN permissions to allow release
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          # @TODO: Add deploy key pair
+          ssh-key: ${{ secrets.DEPLOY_KEY }}
```

Comment:
Noting that this was used to bypass the branch protection rule. But I would like to replace it to use GitHub App in the future. (Specifically, when this actions starts triggering the instance build.)